PotentialFieldControl/obstacle_list: /left_arm/PotentialFieldControl/obstacle
PotentialFieldControl/obstacle_list: /right_arm/PotentialFieldControl/obstacle
root_name: world
tip_name: right_hand_palm_ref_link
topic_desired_reference: desired_reference
topic_obstacle: obstacles
time_interpolation: 5
max_tau_percentage: 0.5
pf_repulsive_gain: 0.1
pf_dist_to_obstacles: 0.15
pf_dist_to_table: 0.15
k_p: [110.0, 110.0, 110.0, 110.0, 110.0, 110.0]
# k_d: [55.0, 55.0, 55.0, 55.0, 55.0, 55.0]
k_d: [10.0, 10.0, 10.0, 10.0, 10.0, 10.0]
# k_i: [2.0, 2.0, 2.0, 2.0, 2.0, 2.0]
k_i: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
# links_with_potential_field: ['right_arm_4_link', 'right_arm_5_link', 'right_arm_6_link', 'right_arm_7_link', 'right_hand_palm_ref_link']
links_with_potential_field: []
enable_joint_limits_avoidance: false
enable_obstacle_avoidance: true
enable_attractive_field: true